frontend/src/Pages/Customer/ViewShops.js

```diff
@@ -66,6 +66,35 @@ class ViewShops extends React.Component {
     });
   }
 
+  convertDegreeToRadians = (latitudeInDegree) => {
+    return Math.PI * latitudeInDegree / 180.0
```

Comment:
Seems like the function is not specific to latitudes, so:
```suggestion
  convertDegreeToRadians = (coordinateInDegree) => {
    return Math.PI * coordinateInDegree / 180.0
```